provisioning/service/src/Query.cs

```diff
@@ -36,25 +36,25 @@ namespace Microsoft.Azure.Devices.Provisioning.Service
     /// On all cases, the <see cref="QuerySpecification"/> contains a SQL query that must follow the
     ///     Query Language for the Device Provisioning Service.
     ///
-    /// Optionally, an <c>Integer</c> with the <b>page size</b>, can determine the maximum number of the items in the
+    /// Optionally, an Integer with the page size, can determine the maximum number of the items in the
     ///     <see cref="QueryResult"/> returned by the <see cref="NextAsync()"/>. It must be any positive integer, and if it
     ///     contains 0, the Device Provisioning Service will ignore it and use a standard page size.
     ///
-    /// You can use this Object as a standard iterator, just using the <c>HasNext</c> and <c>NextAsync</c> in a
-    ///     <c>while</c> loop, up to the point where the <c>HasNext</c> contains <c>false</c>. But, keep
-    ///     in mind that the <see cref="QueryResult"/> can contain a empty list, even if the <c>HasNext</c> contained
-    ///     <c>true</c>. For example, image that you have 10 IndividualEnrollment in the Device Provisioning Service
-    ///     and you created new query with the <c>PageSize</c> equals 5. In the first iteration, <c>HasNext</c>
-    ///     will contains <c>true</c>, and the first <c>NextAsync</c> will return a <c>QueryResult</c> with
-    ///     5 items. After, your code will check the <c>HasNext</c>, which will contains <c>true</c> again. Now,
+    /// You can use this Object as a standard iterator, just using the <code>HasNext</code> and <code>NextAsync</code> in a
```

Comment:
These aren't multi-line so <c> is the correct tag.

Reply:
Although one could argue some of them don't need a code tag at all, like "while" for while loop.